flink/src/main/java/org/apache/iceberg/flink/FlinkTableOptions.java

```diff
@@ -40,4 +40,10 @@ private FlinkTableOptions() {
           .intType()
           .defaultValue(100)
           .withDescription("Sets max infer parallelism for source operator.");
+
+  public static final ConfigOption<Boolean> ENABLE_VECTORIZED_READ =
+      ConfigOptions.key("enable.vectorized.read")
```

Comment:
it seems that Iceberg uses hyphen as config separator. maybe `enable-vectorized-read`?

Reply:
I update it，but I read  the fields in [TableProperties](https://github.com/apache/iceberg/blob/master/core/src/main/java/org/apache/iceberg/TableProperties.java), `-` and  `.` are both being used